src/common/services/session/sessionModal.service.js

```diff
@@ -66,17 +63,9 @@ const SessionModalService = /* @ngInject */ function ($uibModal, $log, modalStat
       openAnalyticsEvent: 'ga-sign-in',
       dismissAnalyticsEvent: 'ga-sign-in-exit'
     }).result,
-    signUp: () => openModal('sign-up').result,
-    forgotPassword: () => openModal('forgot-password').result,
-    resetPassword: () => openModal('reset-password', { backdrop: 'static' }).result,
-    userMatch: () => openModal('user-match', {
-      backdrop: 'static',
-      openAnalyticsEvent: 'ga-registration-match-is-this-you',
-      dismissAnalyticsEvent: 'ga-registration-exit'
-    }).result,
```

Comment:
Are we losing these analytics events? Not seeing this anywhere but the tests now.

Reply:
I've added this back in.